app/Models/User.php

```diff
@@ -31,4 +44,39 @@ class User extends Authenticatable
         'password',
         'remember_token',
     ];
+    /**
+     * Get topics: One to many
+     * @return [type] [description]
+     */
+    public function topics() {
+        return $this->hasMany('App\Models\Topic', 'user_id');
+    }
+    /**
+     * Get posts: One to many
+     * @return [type] [description]
+     */
+    public function posts() {
+        return $this->hasMany('App\Models\Post', 'user_id');
+    }
+    /**
+     * Get comments: One to many
+     * @return [type] [description]
+     */
+    public function comments() {
+        return $this->hasMany('App\Models\Comment', 'user_id');
+    }
+    /**
+     * Get confessions: Many to many
+     * @return [type] [description]
+     */
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Squiz.Functions.MultiLineFunctionDeclaration.BraceOnSameLine</br>**Severity**: error</br>**File**: app/Models/User.php</br>Opening brace should be on a new line